fsw/src/lc_msgdefs.h

```diff
@@ -35,29 +35,37 @@
 #include "lc_platform_cfg.h"
 
 /************************************************************************
- * Macro Definitions
+ * Type Definitions
  ************************************************************************/
 
 /**
  * \name LC Application States
  * \{
  */
-#define LC_STATE_ACTIVE   1 /**< \brief LC Application State Active */
-#define LC_STATE_PASSIVE  2 /**< \brief LC Application State Pasive */
-#define LC_STATE_DISABLED 3 /**< \brief LC Application State Disabled */
-#define LC_STATE_FROM_CDS 4 /**< \brief Used for reset processing, not valid state */
-/**\}*/
+typedef enum LC_AppState
+{
+    LC_STATE_ACTIVE = 1, /**< \brief LC Application State Active */
+    LC_STATE_PASSIVE,    /**< \brief LC Application State Pasive */
+    LC_STATE_DISABLED,   /**< \brief LC Application State Disabled */
+    LC_STATE_FROM_CDS    /**< \brief Used for reset processing, not valid state */
+} LC_AppState_t;
```

Comment:
Type name should be `LC_AppState_Enum_t` and labels should all start with `LC_AppState_` prefix (instead of `LC_STATE_`)

Reply:
Updated.